projects/bootloader/scripts/flash_application_code.py

```diff
@@ -0,0 +1,85 @@
+# pylint: skip-file
+"""This script handles the client side of the flash application code protocol."""
+
+# Import protobuf
+from protogen import flash_application_code_pb2 as pb2
+from can_datagram import Datagram, DatagramListener, DatagramSender
+import can
+import time
+import os
+import sys
+currentdir = os.path.dirname(os.path.realpath(__file__))
+parentdir = os.path.dirname(currentdir)
+sys.path.append(parentdir)
```

Comment:
It's unfortunate due to our hacky project setup, but you'll need to put these lines above the `from protogen import ...` line for python to resolve the imports. (You can test with `make pytest PROJECT=bootloader TEST=flash_application_code`, and you can ignore the pylint errors it'll generate line-by-line.)